src/main/java/com/todolist/app/model/ToDoListAppConstants.java

```diff
@@ -0,0 +1,13 @@
+package com.todolist.app.model;
+
+
+public final class ToDoListAppConstants {
+    /**
+     * This constant represents add button on see-all-tasks page.
```

Comment:
I suppose this constant might be used in other places also, so let's add it more general name, e.g. it represents add operation, because as I see it is being used not only in see-all-tasks servlet but also in newtaskservlet.